sonar-text-plugin/src/main/java/org/sonar/plugins/common/TextAndSecretsSensor.java

```diff
@@ -131,7 +116,7 @@ private static boolean analyzeAllFiles(SensorContext sensorContext) {
   /**
    * In SonarLint context we want to analyze all non-binary input files, even when they are not analyzed or assigned to a language.
    * To avoid analyzing all non-binary files to reduce time and memory consumption in a non SonarLint context only files assigned to a
-   * language are analyzed.
+   * language OR file with a text extension are analyzed.
```

Comment:
text extension so `.txt` files?

Reply:
When I mention `text` file, it's not about `.txt` file, but file that contains human readable data, or non-binary data.
Whle we usually associate such file to the `.txt` extension, it could also be any other file extension/format that are designed to be human readable (`.json`, `.cpp`, ...)
This may be ambiguous, but I couldn't find any alternative/better term, and from my searches, it seems to be the way we express it.

Reply:
Maybe simply `non-binary file`?
Or `file with an extension that from which we can conclude to contain readable text data are analyzed`.
Longer but not so ambiguous

Reply:
non-binary may do the trick, but since we already have a `notBinaryFilePredicate`, the ambiguity may be worse :sweat_smile:
What about `plain-text file`? (`PlainTextFilePredicate`)

Reply:
Would be better.
Looking at the other comment below, I'm not sure we need `TextFilePredicate` afterall